docs/usage.md

```diff
@@ -0,0 +1,99 @@
+# Usage
```

Comment:
Are you planning that we should add other topics to this usage section? It's inconsistent if the pagination docs are separate but the embedded docs are in this usage page. I think that the embedded docs should be in their own sections and we can move the 'Path Customization' and 'coalesceFindRequests option' docs from the README to the usage section.